plugins/providers/bigquery/model.go

```diff
@@ -40,6 +40,8 @@ func (d *Dataset) ToDomain() *domain.Resource {
 		Type: ResourceTypeDataset,
 		Name: d.DatasetID,
 		URN:  fmt.Sprintf("%s:%s", d.ProjectID, d.DatasetID),
+		// urn:{source}:{scope}:{kind}:{identifier}
+		GlobalURN: fmt.Sprintf("urn:bigquery:%s:%s:%s", d.ProjectID, ResourceTypeDataset, d.DatasetID),
```

Comment:
the bigquery urn must be of the format
urn:bigquery:project-01:table:project-01:dataset-01
```suggestion
		GlobalURN: fmt.Sprintf("urn:bigquery:%s:%s:%s:%s", d.ProjectID, ResourceTypeDataset, d.ProjectID, d.DatasetID),
```